legend-engine-server/src/test/java/org/finos/legend/engine/server/test/pureClient/stores/dbSpecific/Relational_DbSpecific_UsingPureClientTestSuite.java

```diff
@@ -39,7 +39,7 @@ public static Test createSuite(String pureTestCollectionPath, String testServerC
         final ThreadLocal<ServersState> state = new ThreadLocal<>();
         state.set(PureTestHelper.initEnvironment(false, testServerConfigFilePath));
 
-        CompiledExecutionSupport executionSupport = getClassLoaderExecutionSupport();
+        CompiledExecutionSupport executionSupport = getClassLoaderExecutionSupport(true);
```

Comment:
why this change?

Reply:
enables console for tests in compiled mode. just to print out some stms like 'server not available' or what error occurred. Rather than just test failed/ assert failed.